CondFormats/CSCObjects/test/testCSCMapping.cpp

```diff
@@ -71,7 +71,7 @@ int testCSCMapping::runIt(const std::string& config){
   edm::AssertHandler ah;
   int rc=0;
   try {
-    edm::EventProcessor proc(config);
+    edm::EventProcessor proc(edm::getPSetFromConfig(config));
```

Comment:
@davidlange6 you miss 

#include "FWCore/ParameterSetReader/interface/ParameterSetReader.h"

Not sure why before it was working with pset

Reply:
Right - it was working before as nothing from ParameterSetReader class is used to define the PSet class structure itself.